src/TerrainRenderer.cpp

```diff
@@ -763,16 +736,21 @@ void TerrainRenderer::Draw(const GameWorldView& gwv, unsigned int* water)
 
     lastOffset = PointI(0, 0);
 
+    // Arrays aktivieren
+    glEnableClientState(GL_VERTEX_ARRAY);
+    glEnableClientState(GL_TEXTURE_COORD_ARRAY);
+    glEnableClientState(GL_COLOR_ARRAY);

```

Comment:
I just made sure all those enabling is done where it belongs and is also disabled after it was enabled. The glInterleaved stuff enabled those in the background which made it non-obvious where things are enabled... There were also some bindings that did not belong there.